src/doc/rustc/src/platform-support/unknown-uefi.md

```diff
@@ -116,7 +116,7 @@ There are 3 common ways to compile native C code for UEFI targets:
 - Use the official SDK by Intel:
   [Tianocore/EDK2](https://github.com/tianocore/edk2). This supports a
   multitude of platforms, comes with the full specification transposed into C,
-  lots of examples and build-system integrations. This is also the only
+  lots of examples and build system integrations. This is also the only
```

Comment:
This is actually correct usage of the hyphen, thus this recommended edit is incorrect. It is an example that follows the model of how [deep-sea fish](https://en.wikipedia.org/wiki/Deep-sea_fish) is written. There, "deep sea" becomes a modifier to "fish". Here, "build system" becomes a modifier to "integrations".
```suggestion
  lots of examples and build-system integrations. This is also the only
```

Reply:
Fixed.